net.christianbeier.Gromit-MPX.yml

```diff
@@ -0,0 +1,31 @@
+id: net.christianbeier.Gromit-MPX
+runtime: org.gnome.Platform # using GNOME for the time being as we need dconf for setting hotkeys under Wayland
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: gromit-mpx
+finish-args:
+  - --socket=x11
```

Comment:
I guess it also works on Wayland natively as it's GTK3?

Reply:
Nope, it depends on XWayland as of now.